packages/bundler/test/BundlerManager.test.ts

```diff
@@ -66,4 +71,94 @@ describe('#BundlerManager', () => {
+        beneficiary: await bundlerSigner.getAddress(),
+        entryPoint: _entryPoint.address,
+        gasFactor: '0.2',
+        minBalance: '0',
+        mnemonic: '',
+        network: '',
+        port: '3000',
+        unsafe: !await isGeth(provider as any),
+        conditionalRpc: false,
+        autoBundleInterval: 0,
+        autoBundleMempoolSize: 0,
+        maxBundleGas: 5e6,
+        // minstake zero, since we don't fund deployer.
+        minStake: '0',
+        minUnstakeDelay: 0
+      }
+      const repMgr = new ReputationManager(BundlerReputationParams, parseEther(config.minStake), config.minUnstakeDelay)
+      const mempoolMgr = new MempoolManager(repMgr)
+      const validMgr = new ValidationManager(_entryPoint, repMgr, config.unsafe)
+      const evMgr = new EventsManager(_entryPoint, mempoolMgr, repMgr)
+      bundleMgr = new BundleManager(_entryPoint, evMgr, mempoolMgr, validMgr, repMgr, config.beneficiary, parseEther(config.minBalance), config.maxBundleGas, false)
+      const execManager = new ExecutionManager(repMgr, mempoolMgr, bundleMgr, validMgr)
+      execManager.setAutoBundler(0, 1000)
+
+      methodHandler = new UserOpMethodHandler(
+        execManager,
+        provider,
+        bundlerSigner,
+        config,
+        _entryPoint
+      )
+    })
+
+    it('should not include a UserOp that accesses the storage of a different known sender', async function () {
+      if (!await isGeth(ethers.provider)) {
+        console.log('WARNING: opcode banning tests can only run with geth')
+        this.skip()
+      }
+
+      const wallet1 = await new TestFakeWalletToken__factory(signer).deploy(entryPoint.address)
+      const wallet2 = await new TestFakeWalletToken__factory(signer).deploy(entryPoint.address)
+
+      await wallet1.sudoSetBalance(wallet1.address, parseEther('1'))
+      await wallet1.sudoSetBalance(wallet2.address, parseEther('1'))
+      await wallet1.sudoSetAnotherWallet(wallet1.address)
+      await wallet2.sudoSetAnotherWallet(wallet1.address)
+
+      const cEmptyUserOp: UserOperation = {
+        sender: AddressZero,
+        nonce: '0x0',
+        paymasterAndData: '0x',
+        signature: '0x',
+        initCode: '0x',
+        callData: '0x',
+        callGasLimit: '0x0',
+        verificationGasLimit: '0x50000',
+        maxFeePerGas: '0x0',
+        maxPriorityFeePerGas: '0x0',
+        preVerificationGas: '0x50000'
+      }
+      const userOp1: UserOperation = {
+        ...cEmptyUserOp,
+        sender: wallet1.address
+      }
+      const userOp2: UserOperation = {
+        ...cEmptyUserOp,
+        sender: wallet2.address
+      }
+      await methodHandler.sendUserOperation(userOp1, entryPoint.address)
+      await methodHandler.sendUserOperation(userOp2, entryPoint.address)
+
+      const bundle = await bundleMgr.sendNextBundle()
+      await bundleMgr.handlePastEvents()
+      const mempool = bundleMgr.mempoolManager.getSortedForInclusion()
+
```

Comment:
1. I reverted the fix, so this test will fail.
2. added here: 
```
      console.log('sender1=', userOp1.sender)
      console.log('sender2=', userOp2.sender)
      console.log((await entryPoint.queryFilter(entryPoint.filters.UserOperationEvent())).map(e=>[e.event, e.args.sender, e.args.nonce, e.args.success]))
```
and I can see that both userops passed and got included, so there is no failure to detect..